docs/DeveloperGuide.md

```diff
@@ -154,6 +154,42 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Add feature
+
+The `add` mechanism is facilitated by `AddressBook`. The implementation of adding a `Candidate` through the `add` command has been enhanced on the existing approach. It extends `Command`. The input parameters of the `add` command has been tweaked slightly, introducing a few more related entities.
```

Comment:
@domlimm I created a PR with the documentation for `find` command! I included a rough layout I thought of to help segment up the different parts to address e.g. _what is the implementation_, _how we implemented it_, _why we decided to implement it that way_ and _relevant UML diagrams_. I'm not sure if it'll be helpful for your part as well - I can update mine accordingly to follow the eventual standardised format! :)

Reply:
I noted on your comment that you'll be updating the PR further - so sorry missed that out earlier! I'll re-look subsequently (if needed) before merging! :sweat_smile:

Reply:
@charmainehly no worries at all! i will take a look at it! thanks char!